modules/ROOT/partials/configuration/premium-upgrade-promotion-option.adoc

```diff
@@ -0,0 +1,28 @@
+== Premium upgrade promotion option
+
+{productname} 6.2 and later includes the `promotion` option. It controls the presentation or otherwise of a Tiny-specific promotion button.
+
+[[promotion]]
+== `+promotion+`
+
+*Type:* `+Boolean+`
+
+*Default value:* `+true+` in Community self-hosted instances; otherwise `+false+`.
+
+*Possible values:* `+true+`, `+false+`
+
+See xref:pages/editor-premium-upgrade-promotion.adoc#premium-upgrade-promotion-defaults[Premium upgrade promotion defaults] for details.
+
+== Example: Using `+promotion+`
+
+[source,js]
+----
+tinymce.init({
+  selector: "textarea",  // change this value according to your HTML
+  promotion: true
+});
+----
+
+The *Upgrade* promotion appears in the unused corner of the {productname} menu bar. Consequently it does not appear if the menu bar is disabled.
+
+NOTE: the distribution-specific defaults are not fixed. Someone running {productname} as part of a Premium plan can turn this option on. And someone running the Community distribution of {productname} as a self-hosted instance can turn this option off.
```

Comment:
```suggestion
NOTE: The distribution-specific defaults are not fixed. Someone running {productname} as part of a Premium plan can turn this option on. And someone running the Community distribution of {productname} as a self-hosted instance can turn this option off.
```
Just because the `NOTE` here is markup not text.